### ISSUE TITLE
Perspective correction controlled through OSC

### ISSUE THREAD

**Issue by ibiltari** [x42/xjadeo#35]
Hi, for an upcoming project (https://github.com/stagesoft) we need this feature and we are planning to implement it on xjadeo, perhaps it could be interesting to include it in the main branch like the other art related features?

**Comment by x42** [comment]
xjadeo does not have any perspective control to begin with, so it cannot be exposed via OSC.

You could probably implement this for the openGL variants in `opengl_draw` (in display_gl_common.h), changing the coordinates of the GL_QUADS.

**Comment by ibiltari** [comment]
Yes thanks!, i'm trying to implement it just like that, modifying the glTexCoord2f of the plane.

I have it working and exposed trough OSC, now im working in the correct texture projection mapping so it can be successfully used to correct perspective from a physical projector. 

My main question was about if you think it could be and interesting addition and i should try to integrate the code with IFDEF's like the other  "art" features so i could generate a pull request when ready and tested. It would be nice for us because we are planning on integrating xjadeo in another open source project and we need this feature.

**Comment by x42** [comment]
That does sound great! PR is definitely welcome.
